docs/platform/howto/enable-backup-to-another-region.rst

```diff
@@ -0,0 +1,167 @@
+Add a backup to another region |beta|
+=====================================
+
+On top of a regular service backup stored in the region where your service is hosted, you can create an additional backup located in a different region. Learn how to enable the backup to another region (BTAR) feature in Aiven.
+
+About enabling BTAR
+-------------------
+
+Tools
+'''''
+
+To add an additional service backup for your service, you can use the following tools:
+
+* `Aiven console <https://console.aiven.io/>`_
+* CLI
+* API
+
+Limitations
+'''''''''''
+
+Currently, BTAR is supported for Aiven for MySQL® and Aiven for PostgreSQL® only.
+
+Prerequisites
+-------------
+
+* Aiven account
+* Depending on the method you choose to use for enabling CCR
+
+  * Access to the `Aiven console <https://console.aiven.io/>`_
+  * `cURL` CLI tool
+  * `Aiven CLI tool <https://github.com/aiven/aiven-client>`_
+
+Back up to another region via console
+-------------------------------------
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select an Aiven service on which you'd like to enable BTAR.
+3. In the **Overview** tab of your service's page, navigate to **Advanced configuration** and select **Change**.
+4. In the **Edit advanced configuration** window, select **Add configuration option** > **additional backup regions**.
+5. Enter a region name into the **additional backup regions** field and select **Save advanced configuration**.
```

Comment:
The `region name` here would not be obvious. 
the user has to actually copy this from some list of regions, (not sure which could be the correct source, but found this https://docs.aiven.io/docs/platform/reference/list_of_clouds)

Reply:
may be a comment indicating where to get that list would be good.